woodwork/table_accessor.py

```diff
@@ -44,18 +45,49 @@ def __init__(self, dataframe):
         self._dataframe_weakref = weakref.ref(dataframe)
         self._schema = None
 
-    def init(self,
-             index=None,
-             time_index=None,
-             logical_types=None,
-             already_sorted=False,
-             schema=None,
-             validate=True,
-             use_standard_tags=True,
-             **kwargs):
-        """Initializes Woodwork typing information for a DataFrame.
+    def init(self, schema=None, **kwargs):
+        """Initializes Woodwork typing information for a DataFrame with only keyword arguments.
+        logical type inference is performed on columns with unspecified logical types"""
+        if schema is not None:
+            warnings.warn("Use init_with_full_schema when passing in a full schema and init_with_partial_schema when passing in a partial schema",
+                          UseInitWithSchemaWarning)
+            self.init_with_full_schema(schema, **kwargs)
```

Comment:
How difficult would it be to determine if a passed in schema was partial or not, and call the appropriate init method accordingly?

Reply:
not very, just threw in a line that checks if all the dataframe columns are in the schema